# How do you assign variants to genes?

Variants sourced from the [Open Targets Genetics Portal](https://docs.targetvalidation.org/data-sources/genetic-associations#open-targets-genetics-portal) are linked to protein-coding genes based on distance within a 1 Mb window of the [lead variant](https://genetics-docs.opentargets.org/faqs#what-is-the-difference-between-lead-variant-and-tag-variant) \(500 kb each side of the lead variant\). The likelihood of a gene being causal for the genetic association represented by the lead variant is captured by the gene prioritisation score. This score is the output of the locus-to-gene pipeline, which integrates fine-mapping, colocalisation and functional genomics.

Variants from other sources than the ones from Open Targets Genetics \(e.g. [European Variation Archive](https://docs.targetvalidation.org/data-sources/genetic-associations#european-variation-archive-eva), [UniProt](https://docs.targetvalidation.org/data-sources/genetic-associations#uniprot), [PheWAS Catalog](https://docs.targetvalidation.org/data-sources/genetic-associations#phewas-catalog)\) are linked to genes and transcripts using the Ensembl Variant Effect Predictor \(VEP\). If a variant maps to a gene with different transcripts, we will assign it the most severe variant definitions or consequence term \(e.g. start loss, stop gained\) from the Sequence Ontology \(SO\). If the variant maps to intergenic regions, we will assign the variant to the 5' end \(i.e. nearest likely promoter\) of the closest protein coding gene on either strand \(forward or reverse\) within a 1 Mb window \(500 kb each side of the variant\).

