# Release notes

## Release 3.21 \(2021-02-17\)

### Status

In our first release of 2021, we integrate 10,078,746 evidence strings to build 7,385,188 target-disease associations between 14,416 diseases and 27,650 targets from the following 19 public resources:

* 794,360 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 388,346 genetic evidence from [Open Targets Genetics Portal](https://genetics.opentargets.org/)
* 2,418 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 20,911 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 1,187 genetic evidence from [ClinGen](https://search.clinicalgenome.org/kb/gene-validity/)
* 183,643 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,269 genetic evidence from [UniProt Literature](https://www.uniprot.org/)
* 10,457 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 3,342 somatic evidence from [intOGen](https://www.intogen.org/)
* 61,666 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 442,053 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 217,307 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 18,393 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,546 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,846 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 710,319 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 6,699,914 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

Key highlights for this release include:

* Data updates from: 
  * ChEMBL
  * ClinGen
  * Europe PMC
  * EVA
  * Gene2Phenotype
  * PhenoDigm
  * Reactome
  * UniProt
* Updates to our [COVID-19 Target Prioritisation Tool](http://covid19.opentargets.org/)
* Preview link to [Beta release of new version of Platform](https://beta.targetvalidation.org/) 

For more details, check our [21.02 release blog post](http://blog.opentargets.org/2021/02/17/open-targets-platform-21-02-release/) and [21.02 technical notes](https://docs.targetvalidation.org/technical-pipeline/technical-notes#21-02).

## Release 3.20 \(2020-11-26\)

### Status

In our fifth and final release of 2020, we integrate 9,220,580 evidence strings to build 6,752,528 target-disease associations between 14,170 diseases and 27,620 targets from the following 19 public resources:

* 793,881 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 388,361 genetic evidence from [Open Targets Genetics Portal](https://genetics.opentargets.org/)
* 2,655 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 20,911 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 1,086 genetic evidence from [ClinGen](https://search.clinicalgenome.org/kb/gene-validity/)
* 183,643 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,200 genetic evidence from [UniProt Literature](https://www.uniprot.org/)
* 10,457 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 3,342 somatic evidence from [intOGen](https://www.intogen.org/)
* 61,666 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 425,873 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 217,309 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 17,346 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,546 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,846 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 680,222 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 5,905,647scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

Key highlights for this release include:

* Implementation of a new scoring method for EVA \(ClinVar\) evidence
* Expanded disease mappings for Genomics England PanelApp evidence
* Data updates from PhenoDigm, PheWAS Catalog, and CRISPR
* Removal of UniProt somatic evidence due to duplication from other somatic evidence sources
* Small change to tractability bucket ordering and categorisation 
* Links to our redesigned profile and association pages

For more details, check our [20.11 release blog post](https://blog.opentargets.org/2020/11/26/open-targets-platform-20-11-release/) and the [20.11 release technical notes](https://docs.targetvalidation.org/technical-pipeline/technical-notes#20-11).



## Release 3.19 \(2020-09-24\)

### Status

In our fourth release of 2020, we integrate 8,419,186 evidence strings to build 6,551,303 target-disease associations between 13,944 diseases and 27,610 targets from the following public resources:

* 107,532 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 390,069 genetic evidence from [Open Targets Genetics Portal](https://genetics.opentargets.org/)
* 2,451 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,556 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 1,075 genetic evidence from [ClinGen](https://search.clinicalgenome.org/kb/gene-validity/)
* 184,013 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,188 genetic evidence from [UniProt Literature](https://www.uniprot.org/)
* 8,173 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 3,342 somatic evidence from [intOGen](https://www.intogen.org/)
* 61,666 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 427,943 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 218,294 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 17,211 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,546 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,844 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 564,310 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 5,912,030 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

New features in this release include:

* New data source - ClinGen
* Data updates from Expression Atlas, ChEMBL, Gene2Phenotype, and Cancer Gene Census
* New chemical probes
* Minor bug fixes

For more details, check our [20.09 release blog post](https://blog.opentargets.org/2020/09/24/open-targets-platform-20-09-release/) and the [20.09 release technical notes](https://docs.targetvalidation.org/technical-pipeline/technical-notes#20-09).

## Release 3.18 \(2020-06-16\)

### Status

In our third release of 2020, we integrate 10,154,924 evidence strings to build 7,282,832 target-disease associations between 13,914 diseases and 27,596 targets from the following public resources:

* 32,195 genetic evidence from [UniProt](http://www.uniprot.org/)
* 116,560 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 390,114 genetic evidence from [Open Targets Genetics Portal](https://genetics.opentargets.org/)
* 2,373 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,554 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 182,694 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,096 genetic evidence from [UniProt Literature](https://www.uniprot.org/)
* 8,227 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 3,324 somatic evidence from [intOGen](https://www.intogen.org/)
* 53,856 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 421,636 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 210,093 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 13,527 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,570 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,844 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 565,233 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 7,640,374 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

New features in this release include:

* Data updates from Expression Atlas, ChEMBL, and PheWAS Catalog
* Updates to the antibody tractability assessments
* Minor bug fixes

For more details, check [our blog](https://blog.opentargets.org/2020/06/17/open-targets-platform-20-06-has-been-released/) and the [technical notes](https://docs.targetvalidation.org/technical-pipeline/technical-notes).

## Release 3.17 \(2020-04-27\)

### Status

In our second release of 2020, we integrate 8,462,444 target-disease associations covering 13,818 diseases and 27,700 targets from the following public resources:

* 32,213 genetic evidence from [UniProt](http://www.uniprot.org/)
* 113,661 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 1,915,083 genetic evidence from [Open Targets Genetics Portal](https://genetics.opentargets.org/)
* 2,375 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,554 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 163,910 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,043 genetic evidence from [UniProt Literature](https://www.uniprot.org/)
* 8,216 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 3,324 somatic evidence from [intOGen](https://www.intogen.org/)
* 53,856 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 400,829 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 204,521 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 13,527 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,570 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,844 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 566,094 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 7,526,568 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

New features include:

* Data updates from Expression Atlas, Reactome, PheWAS Catalog, Gene2Phenotype, and PhenoDigm
* Target safety \(non-clinical experimental toxicity\) data
* Tractability data for modalities other than small molecule and antibody
* Minor bug fixes

For more details, check [our blog](https://blog.opentargets.org/2020/04/27/open-targets-platform-release-20-04-is-out) and the [technical notes](https://docs.targetvalidation.org/technical-pipeline/technical-notes).

## Release 3.16 \(2020-03-02\)

### Status

In the first release of 2020, we integrate 7,999,050 target-disease associations covering 13,445 diseases and 27,717 targets from the following public resources:

* 32,191 genetic evidence from [UniProt](http://www.uniprot.org/)
* 108,100 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 1,911,011 genetic evidence from [Open Targets Genetics Portal](https://genetics.opentargets.org/)
* 1,586 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,551 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 56,255 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,011 genetic evidence from [UniProt Literature](https://www.uniprot.org/)
* 7,837 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 3,324 somatic evidence from [intOGen](https://www.intogen.org/)
* 96,274 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 400,249 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 202,512 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 12,238 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,570 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,844 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 501,788 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 7,251,275 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

New features include:

* Data updates from intOGen and ChEMBL
* Replacement of GWAS Catalog evidence with Open Targets Genetics Portal
* Updated CHEMBL-DrugBank ID mapping, TEP and chemical probes
* Minor bug fixes

For more details, check our [blog](https://blog.opentargets.org/2020/03/02/open-targets-platform-release-20-02-is-out/) and the [technical notes](https://docs.targetvalidation.org/technical-pipeline/technical-notes).

## Release 3.15 \(2019-11-28\)

### Status

In the last release of 2019, we integrate 6,336,307 target-disease associations covering 13,579 diseases and 27,069 targets from the following public resources:

* 31,522 genetic evidence from [UniProt](http://www.uniprot.org/)
* 103,019 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 186,237 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,586 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,550 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 56,255 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,879 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,375 somatic evidence from [intOGen](https://www.intogen.org/)
* 96,274 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 384,754 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 202,512 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,977 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,570 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,844 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 501,788 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 6,856,487 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

New features will include:

* More diseases and revised ontology \(EFO3.11\)
* New pharmacovigilance data and visualisation
* DrugBank ID on drug summary page 
* More chemical probes
* Minor bug fixes throughout the user interface

For more details, check our [blog](https://blog.opentargets.org/2019/11/28/open-targets-platform-release-19-11-is-out/) and the [technical notes](https://docs.targetvalidation.org/technical-pipeline/technical-notes).

## Release 3.14 \(2019-09-24\)

### Status

In this release, we integrate 3,336,588 target-disease associations covering 10,474 diseases and 27,024 targets from the following public resources:

* 35,034 genetic evidence from [UniProt](http://www.uniprot.org/)
* 96,734 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 180,984 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,586 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,528 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 55,987 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,581 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,375 somatic evidence from [intOGen](https://www.intogen.org/)
* 60,310 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 384,173 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 204,229 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,159 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,570 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,659 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 500,683 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 5,772,706 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

New features include:

* More Target Enabling Packages \(TEPs\)
* More data download options
* Minor bug fixes throughout the platform

For more details, check our [blog](https://blog.opentargets.org/2019/09/24/open-targets-platform-release-19-09-is-out/).

## Release 3.13 \(2019-06-27\)

### Status

In this release, we integrate 3,306,780 target-disease associations covering 10,473 diseases and 27,021 targets from the following public resources:

* 33,819 genetic evidence from [UniProt](http://www.uniprot.org/)
* 89,632 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 164,571 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,588 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,533 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 56,014 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,057 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,371 somatic evidence from [intOGen](https://www.intogen.org/)
* 60,371 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 384,814 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 201,881 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,083 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,575 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1,641 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [Behan et al. 2019](https://europepmc.org/abstract/MED/30971826)
* 500,011 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 5,641,475 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

New features include:

* More Target Enabling Packages \(TEPs\) and chemical probes
* More target safety information
* More download options for the drug information table 

For more details, check our [blog](https://blog.opentargets.org/2019/06/27/open-targets-platform-release-19-06-is-out/).

## Release 3.12 \(2019-05-02\)

### Status

In this release, we integrate over 3.3 million target-disease associations covering 10,419 diseases and 28,501 targets from the following public resources:

* 33,310 genetic evidence from [UniProt](http://www.uniprot.org/)
* 89,636 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 157,008 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,589 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,533 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 56,014 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,057 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,371 somatic evidence from [intOGen](https://www.intogen.org/)
* 59,992 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 384,783 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 381,141 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,083 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,575 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 1641 CRISPR-Cas9 \(Cancer Cell Lines\) evidence from [CRISPR](https://score.depmap.sanger.ac.uk/)
* 500,462 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 5,438,280 scientific literature evidence from co-occurence mining in [EuropePMC](https://europepmc.org/)

New features include:

* CRISPR data on evidence page
* Safety data on target profile page
* New TEP data
* Bug fixes throughout the platform

For more details, check our [blog](https://blog.opentargets.org/2019/05/01/open-targets-platform-release-19-04-is-out/).

## Release 3.11 \(2019-02-25\)

### Status

In this release, we integrate over 3.1 million target-disease associations covering 10,183 diseases and 28,426 targets from the following public resources:

* 33,215 genetic evidence from [UniProt](http://www.uniprot.org/)
* 87,170 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 128,960 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,589 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,531 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 54,874 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,061 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,371 somatic evidence from [intOGen](https://www.intogen.org/)
* 58,990 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 284 somatic evidence from [UniProt](http://www.uniprot.org/)
* 476,962 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 379,417 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,085 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,575 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 408 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 505,493 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 5,369,791 scientific literature evidence from co-occurence mining in [EuropePMC](https://europepmc.org/)

New Open Targets Platform features include:

* New data: We have 11% more evidence in release 19.02 when compared to 18.12
* Bug fixes throughout the platform

For more details on the Open Targets Platform releases, check our [blog](http://blog.opentargets.org/2019/02/20/open-targets-platform-release-19-02-is-out/).

## Release 3.10 \(2018-12-14\)

### Status

In this release of the Open Targets Platform we integrate over 3.0 million target-disease associations covering 10,050 diseases and 28,275 targets from the following public resources:

* 31,597 genetic evidence from [UniProt](http://www.uniprot.org/)
* 74,185 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 125,202 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,592 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,531 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 54,875 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,692 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,371 somatic evidence from [intOGen](https://www.intogen.org/)
* 59,028 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 282 somatic evidence from [UniProt](http://www.uniprot.org/)
* 463,578 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 379,325 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,022 affected pathway evidence from [Reactome](https://reactome.org/)
* 74,575 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 406 systems biology evidence from [SysBio](https://saezlab.github.io/progeny/)
* 505,493 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,679,009 scientific literature evidence from co-occurence mining in [EuropePMC](https://europepmc.org/)

New Open Targets Platform features include:

* Improved prioritisation view on disease associations page
* Systems biology data: added datasource to associations facets
* Systems biology data: added display to evidence page
* Bug fixes, UI and data improvements throughout the website

For more details on the Open Targets Platform releases, check our [blog](http://blog.opentargets.org/2018/12/17/new-open-targets-platform-release).

## Release 3.9 \(2018-10-29\)

### Status

In this release of the Open Targets Platform we integrate over 3.0 million target-disease associations covering 10,074 diseases and 21,730 targets from the following public resources:

* 31,220 genetic evidence from [UniProt](http://www.uniprot.org/)
* 74,185 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 112,732 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,596 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 10,531 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 47,302 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,692 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,371 somatic evidence from [intOGen](https://www.intogen.org/)
* 58,001 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 282 somatic evidence from [UniProt](http://www.uniprot.org/)
* 458,207 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 384,424 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,008 affected pathway evidence from [Reactome](https://reactome.org/)
* 89,661 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 505,493 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 5,047,101 scientific literature evidence from co-occurence mining in [EuropePMC](https://europepmc.org/)

New Open Targets Platform features include:

* Target tractability information on target profile page
* Prioritisation view on disease associations page
* Bug fixes, UI and data improvements throughout the website

For more details on the Open Targets Platform releases, check our [blog](https://blog.opentargets.org/2018/10/30/whats-new-in-latest-release).

## Release 3.8 \(2018-08-28\)

### Status

In this release of the Open Targets Platform we integrate over 2.9 million target-disease associations covering 10,101 diseases and 21,149 targets from the following public resources:

* 31,134 genetic evidence from [UniProt](http://www.uniprot.org/)
* 73,805 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 101,551 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,604 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 15,289 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 47,302 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,624 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,371 somatic evidence from [intOGen](https://www.intogen.org/)
* 55,963 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 282 somatic evidence from [UniProt](http://www.uniprot.org/)
* 410,436 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 288,273 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 9,735 affected pathway evidence from [Reactome](https://reactome.org/)
* 89,661 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 465,887 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,906,527 scientific literature evidence from co-occurence mining in [EuropePMC](https://europepmc.org/)

New Open Targets Platform features include:

* Probe Miner information featured on relevant target profile pages
* Odds ratio and confidence information for common diseases evidence
* Bug fixes, UI and data improvements throughout the website

For more details on the Open Targets Platform releases, check our [blog](https://blog.opentargets.org/2018/08/30/our-august-release-is-out/).

## Release 3.7 \(2018-06-25\)

### Status

In this release of the Open Targets Platform we integrate over 2.4 million target-disease associations covering 10,086 diseases and 21,060 targets from the following public resources:

* 30,821 genetic evidence from [UniProt](http://www.uniprot.org/)
* 65,078 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 94,864 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,605 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 14,425 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 47,302 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,391 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,371 somatic evidence from [intOGen](https://www.intogen.org/)
* 55,963 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 282 somatic evidence from [UniProt](http://www.uniprot.org/)
* 399,296 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 289,044 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 9,704 affected pathway evidence from [Reactome](https://reactome.org/)
* 89,654 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 288,471 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,862,965 scientific literature evidence from co-occurence mining in [EuropePMC](https://europepmc.org/)

New Open Targets Platform features include:

* Chemical Probes data featured on relevant target profile pages
* Updated Cancer Biomarkers data table with disease information details
* Updated associations heatmaps visualisation
* Bug fixes, UI and data improvements throughout the website

For more details on the Open Targets Platform releases, check our [blog](http://blog.opentargets.org/2018/06/27/our-release-is-out/).

## Release 3.6 \(2018-05-11\)

### Status

In this release of the Open Targets Platform we integrate over 2.3 million target-disease associations covering 9,700 diseases and 21,016 targets from the following public resources:

* 30,487 genetic evidence from [UniProt](http://www.uniprot.org/)
* 57,619 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 91,608 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,565 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 9,692 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 46,346 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 5,097 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,367 somatic evidence from [intOGen](https://www.intogen.org/)
* 52,533 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 282 somatic evidence from [UniProt](http://www.uniprot.org/)
* 398,227 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 289,298 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 9,760 affected pathway evidence from [Reactome](https://reactome.org/)
* 89,153 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 308 affected pathway evidence from [PROGENy](https://saezlab.github.io/progeny/)
* 288,471 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,718,950 scientific literature evidence from co-occurence mining in [EuropePMC](https://europepmc.org/)

New Open Targets Platform features include:

* Drugs table summary visualisations for quicker overview of drugs data
* Cancer Biomarkers data table
* New data source: PROGENy

API new features and fixes include:

* Fixed batch search issues
* Batch search: added HGNC id lookup
* Search: improved search functionality ngram token length, type and weights

For more details on the Open Targets Platform releases, check our [blog](http://blog.opentargets.org/latest-release-open-targets-platform-is-out/)

## Release 3.5 \(2018-03-12\)

### Status

This release of the Open Targets Platform data includes:

* Targets: 20,974
* Diseases: 9,728
* Evidence: 5,905,247
* Associations: 2,306,670

New features of this release include:

* Cancer Hallmarks data and visualisations
* Bibliography section with 'chips' visualisation and improved sorting
* Association studies with targeted arrays, where only specific sets of variants \(e.g. SNPs\) are genotyped

For more details on the Open Targets Platform releases, check our [blog](http://blog.opentargets.org/open-targets-platform-we-have-a-new-release/)

## Release 3.4 \(2017-12-06\)

### Status

In this release of the Open Targets Platform we integrate over 2.2 million target-disease associations covering 9,294 diseases and 20,633 targets from the following public resources:

* 29,436 genetic evidence from [UniProt](http://www.uniprot.org/)
* 49,496 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 60,245 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,565 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 13,147 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 46,346 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,109 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,367 somatic evidence from [intOGen](https://www.intogen.org/)
* 47,275 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 281 somatic evidence from [UniProt](http://www.uniprot.org/)
* 366,992 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 332,096 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 9,549 affected pathway evidence from [Reactome](https://reactome.org/)
* 89,153 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 288,520 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,441,020 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

This release also includes several new features in the Open Targets Platform website:

* New data and visualisation for mouse phenotypes in the target profile page.
* New table filters in the evidence and profile pages.
* New sections in the drug profile page showing the targets and diseases associated with the drug.
* New section in the drug profile page showing the adverse events of a drug.
* Some bug fixes and small UI improvements throughout the website.

For more details on the Open Targets Platform releases, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 3.3 \(2017-11-03\)

### Status

In this release of the Open Targets Platform we have included two new data sources: somatic mutations from [UniProt](http://www.uniprot.org/) and affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/). In total we integrate over 2.6 million target-disease associations covering 9,341 diseases and 19,803 targets from the following public resources:

* 29,383 genetic evidence from [UniProt](http://www.uniprot.org/)
* 49,366 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 60,150 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 1,565 genetic evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 13,060 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 57,477 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)
* 7,109 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 2,367 somatic evidence from [intOGen](https://www.intogen.org/)
* 38,521 somatic evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 281 somatic evidence from [UniProt](http://www.uniprot.org/)
* 294,635 drug evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 576,133 expression evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,725 affected pathway evidence from [Reactome](https://reactome.org/)
* 89,781 affected pathway evidence from [SLAPenrich](https://saezlab.github.io/SLAPenrich/)
* 511,588 mouse model evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,145,347 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

This release also includes several new features in the Open Targets Platform website:

* New facet in the disease associations page to filter by tissue expression.
* New tissue expression summary in the target profile page.
* New visualisation of target to target relationships based on common associated diseases in the target profile page.
* New visualisation of disease to disease relationships based on common associated targets in the disease profile page.
* New table to display GO terms enrichment in the batch search results.
* Several bug fixes and small improvements throughout the website to improve the usability of the whole website.

For more details on this new release, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 3.2 \(2017-07-27\)

### Status

In this release of the Open Targets Platform we integrate over 2.8 million target-disease associations covering 9,150 diseases and 26,122 targets from the following public resources:

* 30,594 genetic evidence from [UniProt](http://www.uniprot.org/)
* 45,446 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 51,770 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 137,552 evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 310,563 evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,726 affected pathway evidence from [Reactome](http://www.reactome.org/)
* 39,473 evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 2,367 evidence from [intOGen](https://www.intogen.org/)
* 1,566 evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 5,868 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 511,588 evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,145,347 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)
* 6,297 genetic evidence from the [Genomics England PanelApp](https://bioinfo.extge.co.uk/crowdsourcing/PanelApp/)
* 48,660 genetic evidence from the [PheWAS catalog](https://phewascatalog.org/)

Apart from several bug fixes, this release also includes a new table showing the homology relationships for targets in the "Gene tree" section of the target profile page

For more details on this new release, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 3.1 \(2017-05-15\)

### Status

In this release of the Open Targets Platform we integrate over 2.6 million target-disease associations covering 8,891 diseases and 31,380 targets from the following public resources:

* 30,242 genetic evidence from [UniProt](http://www.uniprot.org/)
* 41,005 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 45,009 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 137,648 evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 576,344 evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,726 affected pathway evidence from [Reactome](http://www.reactome.org/)
* 27,738 evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 2,371 evidence from [intOGen](https://www.intogen.org/)
* 975 evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 555 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 395,249 evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 4,060,259 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

This release also includes several bug fixes

For more details on this new release, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 3.0 \(2017-03-30\)

### Status

This release includes several new features in the Open Targets Platform website:

* New tool to search for multiple targets at once
* New web display for protein interactors based on [OmnipathDB data](http://omnipathdb.org/)
* New summary page for Pathways
* New summary page for Drugs
* New FAQs and improvements to the documentation
* More information of the best hit of a search
* Several bug fixes

## Release 2.1 \(2017-03-27\)

### Status

In this release of the Open Targets Platform we integrate over 2.6 million target-disease associations covering 8,718 diseases and 31,352 targets from the following public resources:

* 29,940 genetic evidence from [UniProt](http://www.uniprot.org/)
* 27,846 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 43,004 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 58,860 evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 576,344 evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,692 affected pathway evidence from [Reactome](http://www.reactome.org/)
* 27,738 evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 2,371 evidence from [intOGen](https://www.intogen.org/)
* 975 evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 504 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 395,331 evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 3,952,049 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

This release also includes several bug fixes

For more details on this new release, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 2.0 \(2016-12-09\)

### Status

In this release of the Open Targets Platform we integrate over 2.5 million target-disease associations covering 8,659 diseases and 31,071 targets from the following public resources:

* 27,293 genetic evidence from [UniProt](http://www.uniprot.org/)
* 25,231 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 43,249 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/)
* 123,549 evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/)
* 565,827 evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* 10,642 affected pathway evidence from [Reactome](http://www.reactome.org/)
* 26,914 evidence from the [Cancer Gene Census](http://cancer.sanger.ac.uk/census/)
* 2,371 evidence from [intOGen](https://www.intogen.org/)
* 12,943 evidence from [Gene2Phenotype](https://www.ebi.ac.uk/gene2phenotype)
* 456 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 395,331 evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm)
* 3,751,373 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

Some other highlights include:

* A new redesigned home page
* A new option to filter by _target class_ the targets associated with a disease
* A new feature to filter the targets associated with a given disease based on a user-provided target list
* A new visualisation for GTEx data in the target profile page
* An improved search including drug and phenotype information to retrieve targets and diseases
* A new [Python](http://opentargets.readthedocs.io/en/stable/) client to interface our REST API
* A new page to find our outreach activities
* Improved data retrieval for text mining results
* Many bug fixes and small improvements based on feedback from our users.
* A [survey](https://goo.gl/forms/p7sREp0u2OuJ4nQo1) to get feedback from our users

For more details on this new release, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 1.2.1 \(2016-09-21\)

### Status

In this minor release we have incorporated some data and bug fixes

## Release 1.2 \(2016-08-17\)

### Status

In this third release of the Open Targets Platform, we have new web displays and plenty of new data to assist you in drug discovery and validation:

* 30,591 targets
* 9,425 diseases
* 4.8 million evidence
* 2.4 million target-disease associations

Some highlights:

* New widgets for 'RNA baseline expression' and 'Protein Structure' of a target. In the latter, you can now rotate the protein structure, change its colour, zoom in and out, and highlight any amino acid residue
* Fly and worm are two of the new species in our phylogenetic trees, which have a new name, 'Gene tree'
* You can now find targets in human by searching for their orthologues in fly \(D. melanogaster\) and worm \(C. elegans\), as well as other 12 model organism.
* We have two additional data sources: somatic mutations from IntOGen and variants curated by clinical geneticists from Gene2Phenotype
* You can now export our widgets as a .png file, with scale factors ranging from 1x \(lower resolution\) to 3x \(higher resolution\)
* The options of 'Bubbles', 'Table' and 'Tree' views in the Diseases associated with a target pages are now saved in their URLs
* R client package for Open Targets REST API \(ropentargets\): if you know R, you can now query our API and download all target-disease associations into convenient data frames. ropentargets is available from our Open Targets GitHub repository.
* We have changed the display of the EuropePMC text mining results and made exploration of the "matched sentences" more intuitive
* We have moved to Elasticsearch 2.3.0, which is based on Lucene 5.5.0
* 'Feedback' and 'Follow us' buttons on the homepage: it's so easy to connect with the team now
* We have also updated our help pages, a great place to find out more about the Open Targets Platform

For more details on this new release, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 1.1 \(2016-04-18\)

### Status

This is the second public version of the Open Targets Platform

The Open Targets Platform integrates over 2.1M target-disease associations covering 7,954 diseases and 24,774 targets from the following public resources:

* 9,445 ontology classes from [the Experimental Factor Ontology](http://www.ebi.ac.uk/efo/) \(EFO version 2.69\)
* 10,571 genetic evidence from [UniProt](http://www.uniprot.org/), a comprehensive, high-quality and freely accessible resource of protein sequence and functional information
* 19,857 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/), an open-access database of all types of genetic variation data from human
* 25,536 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/), the NHGRI-EBI Catalog of published genome-wide association studies
* 29,812 evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/), a database that contains compound bioactivity data against drug targets
* 433,809 evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home) that provides information on gene expression patterns under different biological conditions
* 9,468 affected pathway evidence from [Reactome](http://www.reactome.org/), a free, open-source, curated and peer reviewed pathway database
* 9,790 evidence from the [COSMIC cancer Gene Census](http://cancer.sanger.ac.uk/census/), a catalogue of genes for which mutations have been causally implicated in cancer
* 457 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 395,407 evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm), a database of animal model matches ranked by their phenotypic similarity to human disorder
* 3,166,744 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

This release includes the following features described in details in the [documentation](https://github.com/opentargets/docs.targetvalidation.org/tree/7217a4b866df8253ab6f21ceb438f86759a99215/about/README.md):

* REST API: programmatic access to the Open Targets Platform \(see [documentation](https://docs.targetvalidation.org/tutorials/rest-api)\)
* Data dumps: direct access to the data \(see [downloads](https://github.com/opentargets/docs.targetvalidation.org/tree/7217a4b866df8253ab6f21ceb438f86759a99215/dumps/README.md)\)
* Look and Feel: new 'Open Targets' logo and new color palette
* Search results: results of a search are presented by order of relevance and indicates the number of diseases associated for any target and the number of targets associated for any disease. Matched terms are highlighted
* Target Association page: Therapeutic Areas are now presented as individual facets that can be combined. This is consistent with the data types facets
* Target-Disease Evidence page: somatic mutations from the Cancer Gene Census are presented by mutation type with the number of samples recorded for each mutation type. The cellular mechanism of the mutation \(dominant/recessive\) is also given based on the Cancer Gene Census annotation
* Target-Disease Evidence: for known drug evidence, the clinical trial phase status is presented

For more details on this new release, check our [blog](https://blog.opentargets.org/tag/release-notes)

## Release 1.0 \(2015-12-08\)

### Status

This is the first public version of the CTTV platorm

The CTTV platform integrates over 2M target-disease associations covering 7,359 diseases and 24,309 targets from the following public resources:

* 9,600 ontology classes from [the Experimental Factor Ontology](http://www.ebi.ac.uk/efo/) \(EFO version 2.66\)
* 9,741 genetic evidence from [UniProt](http://www.uniprot.org/), a comprehensive, high-quality and freely accessible resource of protein sequence and functional information
* 19,084 genetic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/), an open-access database of all types of genetic variation data from human
* 25,876 genetic evidence from [GWAS Catalog](https://www.ebi.ac.uk/gwas/), the NHGRI-EBI Catalog of published genome-wide association studies
* 20,830 evidence from [ChEMBL](https://www.ebi.ac.uk/chembl/), a database that contains compound bioactivity data against drug targets
* 377,360 evidence from [Expression Atlas](https://www.ebi.ac.uk/gxa/home) that provides information on gene expression patterns under different biological conditions
* 9,390 affected pathway evidence from [Reactome](http://www.reactome.org/), a free, open-source, curated and peer reviewed pathway database
* 12,079 evidence from the [COSMIC cancer Gene Census](http://cancer.sanger.ac.uk/census/), a catalogue of genes for which mutations have been causally implicated in cancer
* 459 somatic evidence from [European Variation Archive \(EVA\)](http://www.ebi.ac.uk/eva/)
* 440,601 evidence from [PhenoDigm](https://www.sanger.ac.uk/resources/databases/phenodigm), a database of animal model matches ranked by their phenotypic similarity to human disorder
* 3,327,695 scientific literature evidence from co-occurence mining in [Europe PMC](https://europepmc.org/)

This release includes the following features described in details in the [documentation](https://github.com/opentargets/docs.targetvalidation.org/tree/7217a4b866df8253ab6f21ceb438f86759a99215/about/README.md):

* Search box: you can retrieve targets of interest by entering gene symbols, gene names, gene synonyms, Ensembl identifiers and UniProt accessions
* Search box: you can also search by disease name provided the disease label and its synonyms are part of the Experimental Factor Ontology \(EFO\)
* Search box: you can also search by phenotype provided the phenotype label and its synonyms are included in the Experimental Factor Ontology \(EFO\)
* Search results: results of a search are presented by order of relevance based on the matching of symbols, names and synonyms. Matched terms are highlighted
* Search filters: search results can be filtered by target and/or disease. Number of matching targets and number of matching diseases are shown
* Auto-completion box: search terms are auto-completed. the auto-completion presents - up to 3 - matched target and disease terms
* Auto-completion box: EFO disease ontology classification is presented under the disease term and clickable. A disease hyperlink links to the disease association page
* Auto-completion box: the hyperlink of an auto-completed target links to a Target Association page for this target
* Target Association page: provides an overview of all diseases associated to a target. 3 views are available: a 'bubble view', a 'tabular view' and a 'tree' view
* Target Association page bubble view: groups disease by therapeutic area into bubbles. Because of the EFO ontology, a disease can belong to multiple therapeutic area
* Target Association page bubble view: you can click on any bubble to get the association score between a specific disease and a target. This score is broken down by data types and represented as a flower
* Target Association page bubble view: you can go to a detailed target-disease Evidence page by following the view details link when clicking on a bubble
* Target Association page tabular view: lists all diseases associated to a particular gene ordered by association score. This score is broken down by data types
* Target Association page tree view: represents diseases associated to a specific target at different classification levels of the EFO ontology. Nodes in the tree can be collapsed or expanded
* Target Association page filters: filter by datatypes and therapeutic areas
* Disease Association page: provides a tabular view of all potential targets associated to a disease ordered by association score. This score is broken down by data types
* Target-Disease Evidence page: provides details on the evidence found to associate a particular target to a particular disease \(from EFO\). The association score flower represents the relative contribution of each data type when scoring a target to disease association
* Target-Disease Evidence: genetic associations evidence from eVA, UniProt and the GWAS catalog are presented. A genome browser is embedded to display mutations
* Target-Disease Evidence: somatic mutations evidence from the cancer gene census COSMIC resource and from eVA are presented
* Target-Disease Evidence: known drug evidence from ChEMBL are presented
* Target-Disease Evidence: RNA expression evidence from the Expression Atlas are presented
* Target-Disease Evidence: target-disease co-occurence evidence mined from Europe PMC are presented
* Target-Disease Evidence: disrupted pathways evidence from Reactome are presented
* Target-Disease Evidence: mouse models of disease from PhenoDigm are presented
* Target Profile page: summarises all information known about a particular target including synonyms, UniProt and Ensembl information, GO terms, baseline protein and RNA expression and list of publications from EUROPE PubMed Central
* Disease Profile page: provides a description of a disease, its synonyms and its classification in the Experimental Factor Ontology

The CTTV platform also integrates third-party visualisations components:

* The human baseline expression widget from the EMBL-EBI [Expression Atlas](https://www.ebi.ac.uk/gxa/home)
* The protein features widget from [UniProt](http://www.uniprot.org/)
* The pathway navigation widget from [Reactome](http://www.reactome.org/)
* The protein structure visualisation is extracted from [PDBe](https://www.ebi.ac.uk/pdbe/)

