# Target profile

When searching for a target, in addition to the list of diseases associated with it, you will also see at the top of the page the ‘View TARGET profile’ link e.g. 'View BRAF profile'. Click on the link to get information on your target of interest, outside the context of associated diseases. 

In the target profile page, you will see the target official symbol, its description and synonyms, the latter two provided by UniProt. Scroll down to see detailed information on the target \(e.g. Drugs, Protein interactions\) and the original resource of the data \(e.g. ChEMBL, OmniPath DB\):

* [Target safety](https://docs.targetvalidation.org/getting-started/target-safety): manual curation of Bowes et al. \(2012\), Urban et al. \(2012\), Lamore et al. \(2017\), Lynch et al. \(2017\),  Force and Kolaja \(2011\), HeCaToS, eTOX, and Tox21 

{% hint style="success" %}
Target safety data is subdivided into:

* Known safety effects
* Safety risk information
* Non-clinical experimental toxicology

This data summarises toxic effects if/when modulating a given target and affected organs \(or systems\) reported in the sources listed. Organs were mapped to the Uber-anatomy ontology.

This data is currently available for 439 targets, such as [TBXA2R](https://www.targetvalidation.org/target/ENSG00000006638) and [ADORA2A](https://www.targetvalidation.org/target/ENSG00000128271).
{% endhint %}

* Drugs: [ChEMBL](https://www.ebi.ac.uk/chembl/)
* Target Tractability: [ChEMBL](https://www.ebi.ac.uk/chembl/)
* Protein information: [UniProt](http://www.uniprot.org/)
* Pathways: [Reactome](http://www.reactome.org/)
* Similar targets \(based on diseases in common\)
* Variants, isoforms and genomic context: [Ensembl](https://legacy.gitbook.com/book/opentargets/docs/edit#)
* Protein interactions: [OmniPathDB](http://omnipathdb.org)
* RNA and protein baseline expression: [Expression Atlas](https://www.ebi.ac.uk/gxa/home), [Human Protein Atlas](http://www.proteinatlas.org/), [GTEx](https://www.gtexportal.org/home/documentationPage) 
* Mouse phenotypes: [MGI](http://www.informatics.jax.org)
* Protein structure: [PDBe](https://www.ebi.ac.uk/pdbe/)
* Gene ontology: [Gene Ontology Consortium](http://geneontology.org/) via [UniProt](http://www.uniprot.org/)
* Gene tree: [Ensembl Protein trees and orthologies](http://www.ensembl.org/info/genome/compara/homology_method.html)
* Bibliography: [Open Targets Library](https://github.com/opentargets/docs.targetvalidation.org/tree/89163f4f81591e3fa4ca1b7890ee9e8f49c7e22b/library.opentargets.io)
* Cancer hallmarks: [Cancer Gene Census](https://cancer.sanger.ac.uk/census#cl_search) - COSMIC
* Cancer biomarkers: [Cancer Biomarkers database](https://www.cancergenomeinterpreter.org/biomarkers)

We also include the following data to assist prioritisation of drug targets, whenever it is available:

* [Target Enabling Packages](http://www.thesgc.org/tep) \(TEPs\) from the [Structural Genomics Consortium](https://www.thesgc.org/chemical-probes) \(SGC\) e.g. [CDK12](http://www.targetvalidation.org/target/ENSG00000167258)
* High-quality chemical probes from the SGC, the [Chemical Probes Portal](http://www.chemicalprobes.org/) and/or [Open Science Probes](http://www.sgc-ffm.uni-frankfurt.de/) e.g. [BRD4](https://www.targetvalidation.org/target/ENSG00000141867)
* Potential chemical probes for several targets from [Probe Miner](https://probeminer.icr.ac.uk/) e.g. [SCN5A](https://www.targetvalidation.org/target/ENSG00000183873), 
* CRISPR-Cas9 cancer cell line dependency data for several targets from [Project Score](https://score.depmap.sanger.ac.uk/) e.g. [RHBDF1](https://www.targetvalidation.org/target/ENSG00000007384)

{% hint style="info" %}
Check our [Everything you wanted to know about a drug target](https://www.youtube.com/playlist?list=PLncWVtwSXtqb2kA4gGVJUrbY4baD3iFDx) series of quick tutorials on YouTube to learn how to explore some of the annotation in the target profile page.
{% endhint %}

{% embed url="https://www.youtube.com/watch?v=INdoXk4jQ4E&list=PLncWVtwSXtqb2kA4gGVJUrbY4baD3iFDx" %}

